finals-2024-01-15/code/5105-angela-cekovska/src/app/about/about.component.html

```diff
@@ -0,0 +1,12 @@
+<div>
+
+    <h2>About Page</h2>
+    <ul>
+
+        <li>Name: {{ studentName }}</li>
+        <li>ID: {{ studentID }}</li>
+        <li>Current Year: {{ current year }}</li>
```

Comment:
There's an unfortunate space here (`current year` instead of `currentYear`) that prevented the app from being run. +3 points for the build status